deploy/compliance-operator/config.yaml

```diff
@@ -5,5 +5,5 @@ selectorSyncSet:
     operator: In
     values: ["staging"]
   matchLabels:
-    # Only apply on clusters owned by "Red Hat OpenShift Online" (SREP)
-    api.openshift.com/legal-entity-id: "1HELaFOf2YHWwwvt3XMbT5Mja7M"
+    # Only apply on clusters owned by (SREP)
+    api.openshift.com/legal-entity-id: "${{SREP_LEGAL_ENTITY_ID}}"
```

Comment:
Does it need two braces? I don't see that here: https://github.com/openshift/managed-cluster-config/blob/503a50b9da8301ddd76106a70520e4d58a0b56c2/deploy/cluster-monitoring-config/cluster-monitoring-config.yaml#L45

Reply:
I took insipiration from:
https://github.com/openshift/managed-cluster-config/blob/master/deploy/sre-authorization/02-osd-sre-admins.Group.yaml#L5
```
users: "${{SREP_USERS}}"
```

I did see the line you mentioned but it was inside and inline configmap :man_shrugging: might just still work but went with SREP group example as I am more familiar with it.